.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Issue template
+about: Create a new issue for `Woof` project.
+title: ''
+labels: need more discussion
+assignees: ''
+
+---
+
+<!-- This references to the issues that block the current issue-->
+## Depends on tasks:
+- #_issue
+- #_issue
+
+<!-- Provides the general description of the issue-->
+## Description
+
+<!-- Describes the requirements from the user's point of view-->
+### User requirements:
+
```

Comment:
Do we need to add:

## Acceptance Criteria:

Reply:
Added this section before user story and dev restrictions